jobba.gemspec

```diff
@@ -19,12 +19,11 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "redis", "~> 3.2"
-  spec.add_runtime_dependency "oj", '~> 3.7.6'
+  spec.add_runtime_dependency "redis"
+  spec.add_runtime_dependency "oj"
   spec.add_runtime_dependency "redis-namespace"
 
-  spec.add_development_dependency "bundler", "~> 1.10"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake"
   spec.add_development_dependency "rspec"
```

Comment:
these make me nervous, but it should be fine since we have good spec coverage and it's only used internally.  If we had a lot of open-source users, I'd be afraid of what kind of bug reports we'd get.   If we wanted to still have some checks, maybe we could do '>= 2.0' or whatever the minimum we want to support

Reply:
Should be fine for gems, we lock them more specifically in the apps themselves.